woocommerce/class-sv-wc-plugin.php

```diff
@@ -1454,6 +1460,17 @@ protected function set_dependencies( $dependencies = [] ) {
 		wc_deprecated_function( __METHOD__, '5.2.0' );
 	}
 
+	/**
+	 * Add the HPOS compatibility.
+	 *
+	 * @since x.y.z
+	 */
+	public function add_hpos_compatibility() {
+
```

Comment:
because of this bug https://godaddy.slack.com/archives/C026JPZ4GLB/p1673409266503829 it's quite possible that we will need additional logic in this method - for instance check if the installed WC version is the one that contains the bugfix